python/drned_xmnr/op/config_op.py

```diff
@@ -68,6 +68,29 @@ def perform(self):
                 str([self.state_filename_to_name(st) for st in state_files])}
 
 
+class ViewStateOp(ConfigOp):
+    action_name = 'view state'
+
+    def _init_params(self, params):
+        self.state_name = params.state_name
+
+    def perform(self):
+        self.log.debug("config_view_state() with device {0}".format(self.dev_name))
+        state_name = self.state_name
+        state_filename = self.state_name_to_filename(state_name)
+        if state_filename == []:
+            raise ActionError('no such state: ' + self.state_name)
+        try:
+            f = open(state_filename, 'r')
```

Comment:
I suggest you use the idiom `with open(..) as f:` - this way you make sure everything is cleaned up on completion/failure.